src/component/absensi/shift/NewShift.vue

```diff
@@ -1,22 +1,22 @@
 <template>
-    <div class="container">
-        <h2>New Shift</h2>
-        <form method="post">
-            <div class="form--section">
-                <label for="shiftName"> Shift Name  </label>:
-                <input type="text" name="shiftName" id="shiftName">
+    <div class="newShift">
+        <h2 class="newShift__title">New Shift</h2>
+        <form method="post" class="newShift__form">
+            <div class="newShift__form__section">
+                <label for="shiftName" class="newShift__form__section__label"> Shift Name</label>
+                <input type="text" name="shiftName" id="shiftName" class="newShift__form__section__input">
             </div>
-            <div class="form--section">
-                <label for="startShift"> Shift Start  </label> :
-                <input type="time" name="startShift" id="startShift">
+            <div class="newShift__form__section">
+                <label for="startShift" class="newShift__form__section__label"> Shift Start</label>
+                <input type="time" name="startShift" id="startShift" class="newShift__form__section__input">
             </div>
-            <div class="form--section">
-                <label for="endShift"> Shift End  </label>:
-                <input type="time" name="endShift" id="endShift">
+            <div class="newShift__form__section">
+                <label for="endShift" class="newShift__form__section__label"> Shift End</label>
+                <input type="time" name="endShift" id="endShift" class="newShift__form__section__input">
             </div>
-            <div class="form--section">
-                <label for="department"> Department</label>:
-                <select name="department" id="department" >
+            <div class="newShift__form__section">
+                <label for="department" class="newShift__form__section__label"> Department</label>
```

Comment:
I changed it into `new-shift__label`, `new-shift__input` and `new-shift__section`